sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -186,6 +186,194 @@ T fabs(T x);
 
 Compute absolute value of a `bfloat16`.
```

Comment:
I know you didn't add this description of `fabs` in this PR, but the wording is strange because it doesn't match the wording you added for all the new functions.  Does `fabs` support `sycl::marray<bfloat16, N>` also?  If so, we should change the description to say that.  If it does not, why not?

Reply:
The fabs also supports sycl::marray<bfloat16, N>, have updated the description.
Thanks very much.